11ty/definitions/blacklist-whitelist.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Blacklist Whitelist
+slug: blacklist-whitelist
+flag:
+  level: avoid
+  text: racist term
+defined: true
+exceprt: terms used in many contexts to represent a list of entities or people who are to be allowed or denied access to a service, privilege, or recognition. To "blacklist" is "to exclude", to "whitelist" means "to include".
+speech: noun; verb
+alt_words:
+  - 'blacklist (noun): blocklist, denylist, excludelist'
+  - 'whitelist (noun): allowlist, passlist, trustlist, safelist'
+  - 'blacklist (verb): block, deny, exclude'
+  - 'whitelist (verb): allow, trust, include'
+reading:
+  - text: Basecamp stops using black/whitelist (tweet)
+    href: https://twitter.com/dhh/status/1032050325513940992
+  - text: Google Chrome team moves away from black/whitelist to be more inclusing
+    href: https://9to5google.com/2020/06/12/google-android-chrome-blacklist-blocklist-more-inclusive/
+  - text: Google developers styleguide entry for blacklist
+    href: https://developers.google.com/style/word-list#blacklist
+---
+
+To allow or deny access to a service, a privilege, or recognition.
+
+## Issue
+
+Using a color to denote whether an entity is included or excluded from a group draws parallels to the idea that Black and White people have respectively been denied and allowed privileges based on the color of their skin. Similarly to the concept of [master/slave](/definitions/master-slave), blacklist/whitelist is used casually in the realm of software development, which reinforces the exclusion of Black people in the industry.
+
```

Comment:
Under this area, I'd love a bit more finessing and context added with the historical context and origins of the term:

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148600/ offers some tactical insights about where the terms first began and the context of slavery.

I suggest that we parse the issues into two main buckets of issues:
- Black/white as a good/bad dichotomy
- Black/white admittance and denial (Jim Crow)